### ISSUE TITLE
Allow aria-required on password input element

### ISSUE THREAD

**Issue by 2biazdk** [w3c/aria#1461]
`<input type="password">` has no role (https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings) and therefore only allows global attributes (https://www.w3.org/TR/wai-aria-1.2/#global_states) which doesn't contain `aria-required`.

As far as I can see, `aria-required` should be allowed on the password input element.

**Comment by scottaohara** [comment]
it has no implicit ARIA role mapping, but that doesn't mean it doesn't allow other non-global `aria-*` roles.

see [ARIA in HTML](https://w3c.github.io/html-aria/#el-input-password)

**Comment by Jym77** [comment]
Ouch! This means we've completely messed up ACT rule [ARIA state or property is permitted](https://act-rules.github.io/rules/5c01ea) which completely ignores the ARIA in HTML and requires that elements with no role only have the global attributes 😭
